src/trajectory/Interpolated.cpp

```diff
@@ -100,6 +100,17 @@ void Interpolated::evaluateDerivative(double _t, int _derivative,
     _tangentVector.setZero();
   }
 
+  if(_t==mWaypoints[0].t)
```

Comment:
@mkoval When _t == mWaypoints[0].t, `int idx == getWaypointIndexAfterTime(_t)` returns 0 to idx. Then a domain error will be thrown. I needed the velocity of the start time of an `Interploated` to convert it to a parabolic cubic `Spline`. Since you suggested to convert it into a piecewise linear `Spline`, I think I do not need the velocities any more. 

Since we are not currently supporting the time before the start time now. As you pointed out previously that it is not continuous here thus no derivative shall be supported. Maybe I could just revert all the changes made here. Does this make sense?